### PR TITLE
Creation of the styles variables used in the project + adding logos i…

### PR DESCRIPTION
**Pull request by MatteiT**
Test PR a voir avec Ban

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,6 +7,14 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link rel="icon" type="image/x-icon" href="favicon.ico" />
     <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Rubik:ital,
```
Comment (Tylyfox) [inline src/index.html:13]: indentation?

File: src/styles.scss [hunk 2]
```diff
@@ -1,4 +1,43 @@
...
-html, body { height: 100%; }
-body { margin: 0; font-family: Roboto, "Helvetica Neue", sans-serif; }
+// Définition des variables pour les couleurs
+$main-color: #F67280 ;
+$secondary-color: #C06C84 ;
+$tertiary-color: #6C5B7B ;
+$quaternary-color: #355C7D ;
+
+// Définition des variables pour les fonds
+
```
Comment (Tylyfox) [inline src/styles.scss:9]: retour a la ligne? xD

File: src/styles.scss [hunk 3]
```diff
@@ -1,4 +1,43 @@
...
+// Définition des poids de polices Rubik
+
+$font-weight-light: 300;
+$font-weight-normal: 400;
+$font-weight-medium: 500;
+$font-weight-bold: 700;
+
+// Définition des variables pour les tailles de polices
+html, body { height: 100%; font-family: $font; margin: 0;
+}
```
Comment (Tylyfox) [inline src/styles.scss:32]: l'accolade ne doit ps être au dessus?

**Review by Tylyfox — CHANGES_REQUESTED** [review]

**Review by Benja-san — CHANGES_REQUESTED** [review]

File: src/styles.scss [hunk 4]
```diff
@@ -1,4 +1,79 @@
...
+
+// Définition des variables pour les tailles de polices
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: $font;
+  font-size: 16px;
+  color: $text-color;
+  background-color: $bg-color;
```
Comment (Benja-san) [inline src/styles.scss:39]: Does all your elements need to have a background color ? This one could remain on the body I guess ?

**Review by Benja-san — CHANGES_REQUESTED** [review]

File: src/styles.scss (hunk 4 above)
Comment (MatteiT) [reply src/styles.scss]: nope , changed in new push

**Review by Benja-san — APPROVED** [review]

**Review by Tylyfox — APPROVED** [review]
I approve very good! Nice job Tristan!!

**Pull request merged**